packages/base/card-api.gts

```diff
@@ -2053,8 +2053,10 @@ function makeDescriptor<
     return field.getter(this);
   };
   if (field.computeVia) {
-    descriptor.set = function () {
-      // computeds should just no-op when an assignment occurs
+    descriptor.set = function (this: CardInstanceType<CardT>, value: any) {
+      value = field.validate(this, value);
+      let deserialized = getDataBucket(this);
+      deserialized.set(field.name, value);
```

Comment:
this change was necessary for being able to deserialize not-loaded computed linksTo + linksToMany -- uncovered in serialization tests. otherwise throws error and blows up.

Reply:
is the idea here that we need to be able to replace the not-loaded value with the actual value after it's ready?

Reply:
is there a reason why we don't set the value directly using the deserialized data bucket? i think that's how we deal with this for the non computed fields.

Reply:
> is the idea here that we need to be able to replace the not-loaded value with the actual value after it's ready?

Yes, I think that's what we do for the other not-loaded fields

Reply:
no i mean keeping this set property descriptor as no op. but in the place that you do need to set the computed field's value, don't set it using `instance.fieldName = 'xxxx'`, but set it using the exact way that you added here. The idea is that we don't want the outside world to be able to set computeds, but from inside the card we can do the field value assignment using the deserialized bucket directly.